packages/aws-cdk/lib/api/stack-events/stack-activity-monitor.ts

```diff
@@ -119,12 +118,12 @@ export class StackActivityMonitor {
 
   public async start() {
     this.monitorId = uuid.v4();
-    await this.ioHelper.notify(debug(`Deploying ${this.stackName}`, 'CDK_TOOLKIT_I5501', {
```

Comment:
debug to info? intentional?

Reply:
Yes, they are `info` from the toolkit perspective. In our CLI we handle the Stack Monitoring messages differently (via the activity printers `--progress=events` and `--progress=bar`), so we don't actually care about the levle here.